src/models/Student.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require('mongoose');
+const User = require('./User');
+const {
+  adultValidator,
+  adultNameFirstAndLast,
+} = require('../utils/adultValidation');
+const { lettersOnlyValidator } = require('../utils/letterValidation.js');
+
+const StudentSchema = new mongoose.Schema(
+  {
+    adultName: {
+      type: String,
+      validate: [
+        lettersOnlyValidator,
+        ...adultValidator,
+        adultNameFirstAndLast,
+      ],
+    },
+    phoneNumber: {
+      type: String,
+      default: '',
+      match: [
+        /^(?:\+?\d{1,3}[-.\s]?)?(?:\(?\d{1,4}\)?[-.\s]?)?\d{1,4}[-.\s]?\d{1,4}[-.\s]?\d{1,9}$/,
+        'Please provide a valid phone number',
+      ],
+    },
+    dateOfBirth: {
```

Comment:
validate: {
    validator: function(value) {
      return value < Date.now();
    },
    message: 'Date of birth must be in the past',
  },

Reply:
Added, Thank you!